features/waterOrdering/replies.js

```diff
@@ -10,7 +10,16 @@ module.exports = {
     'Done :white_check_mark:.',
     'Yes!',
   ],
-  sendingError: [
+  askRespect: [
+    'Maybe "Please"? :grumpy:',
+    'You ask me for a service, but do it without respect? :godfather:',
+    'Actually, I deserve respect :dude:',
+    'How dare you not respect me? :accepted:',
+    'Water shall not pass!!! :not_pass:',
+    'Demand respect I :yoda:',
+  ],
+  respectNoteText: '_(you can show you respect with *please*, *pls* or *plz* at the end of message)_',
```

Comment:
Thoughts about making it simpler?

`respectful requests include *please*, *pls* or *plz*`